frb_example/pure_dart/dart/lib/main.dart

```diff
@@ -408,6 +408,13 @@ void main(List<String> args) async {
     expect(sum, equals(21 + 1 + 5));
   });
 
+  test('return SumWith array test', () async {
+    final List<SumWith> sumWithList = await api.getSumArray(a: 12, b: 23, c: 67);
+    expect(sumWithList[0].sum(y: 23, z: 67), 12 + 23 + 67);
+    expect(sumWithList[0].sum(y: 12, z: 67), 12 + 23 + 67);
+    expect(sumWithList[0].sum(y: 12, z: 23), 12 + 23 + 67);
```

Comment:
nit: I guess you mean [0], [1], [2]?

Reply:
Yep...

Reply:
Hey it is still `[0]` not [0] [1] [2]...

Reply:
Sorry, IDK how I missed it again. I think after trying to merge the master, deleted the commit or un committed code...